src/components/Proposal/ActionButton.tsx

```diff
@@ -113,10 +113,9 @@ class ActionButton extends React.Component<IProps, IState> {
       beneficiaryHasRewards = Object.keys(contributionRewards).length > 0;
     }
 
-    const redeemable = proposalState.executedAt
-                       && (accountHasGPRewards
-                           || (proposalState.winningOutcome === IProposalOutcome.Pass && beneficiaryHasRewards));
-
+    const redeemable = proposalState.executedAt && proposalState.accountsWithUnclaimedRewards.includes(currentAccountAddress.toLowerCase());
+    // hack to work around https://github.com/daostack/subgraph/issues/304
+    const reallyRedeemable = redeemable && (beneficiaryHasRewards || accountHasGPRewards);
```

Comment:
what about the proposalState.winningOutcome === Pass part? that fixed a bug recently where proposals thought they could be redeemed when they couldn't because they expired